src/test/java/com/google/cloud/tools/managedcloudsdk/install/DownloaderTest.java

```diff
@@ -99,7 +100,9 @@ public void testDownload() throws IOException, InterruptedException {
     Assert.assertArrayEquals(Files.readAllBytes(destination), Files.readAllBytes(testSourceFile));
 
     ProgressVerifier.verifyProgress(
-        mockProgressListener, "Downloading " + String.valueOf(testFileSize) + " bytes");
+        mockProgressListener,
+        String.format(
```

Comment:
This should be a string literal, not a simple replication of the logic. See the recent ToTT episode on keeping logic out of tests: https://big.corp.google.com/~jmcmaster/testing/2014/07/episode-338-dont-put-logic-in-tests.html

Reply:
True, should be fixed now.